modules/core/src/main/scala/parser.scala

```diff
@@ -189,7 +189,7 @@ object GraphQLParser {
     QueryShorthand.widen[Ast.OperationDefinition] | Operation.widen
 
   lazy val QueryShorthand: Parser[Ast.OperationDefinition.QueryShorthand] =
-    SelectionSet.map(Ast.OperationDefinition.QueryShorthand)
+    SelectionSet.map(Ast.OperationDefinition.QueryShorthand.apply)
```

Comment:
Scala 3 warns that `apply`-insertion is deprecated. I guess the companions no longer extend function types?